src/tests/conduit/t_conduit_node.cpp

```diff
@@ -1162,3 +1162,60 @@ TEST(conduit_node, add_child)
 
 
 
+//-----------------------------------------------------------------------------
+TEST(conduit_node, describe)
+{
+    Node n;
+
+    n["a"] = {1,2,3,4,5};
+    n["b"] = {1,2,3};
+    n["c"] = {1,2,3,4,5,6};
+    n["d"] = {1,2,3,4,5,6,7};
+    n["e"] = {1,2,3,4,5,6,7,8,9,10,11,12};
+    n["f"] = {1.0,2.0,3.0,4.0,5.0,6.0,7.0};
+    n["g"] = {2.0,4.0};
+
+    Node d;
+    n.describe(d);
+    d.print();
+
+    EXPECT_EQ(d["a/count"].to_int(),5);
+    EXPECT_EQ(d["b/count"].to_int(),3);
+    EXPECT_EQ(d["c/count"].to_int(),6);
+    EXPECT_EQ(d["d/count"].to_int(),7);
+    EXPECT_EQ(d["e/count"].to_int(),12);
+    EXPECT_EQ(d["f/count"].to_int(),7);
+
+    EXPECT_EQ(d["a/min"].to_int(),1);
+    EXPECT_EQ(d["b/min"].to_int(),1);
+    EXPECT_EQ(d["c/min"].to_int(),1);
+    EXPECT_EQ(d["d/min"].to_int(),1);
+    EXPECT_EQ(d["e/min"].to_int(),1);
+    EXPECT_EQ(d["f/min"].to_float(),1.0);
+
+    EXPECT_EQ(d["a/max"].to_int(),5);
+    EXPECT_EQ(d["b/max"].to_int(),3);
+    EXPECT_EQ(d["c/max"].to_int(),6);
+    EXPECT_EQ(d["d/max"].to_int(),7);
+    EXPECT_EQ(d["e/max"].to_int(),12);
+    EXPECT_EQ(d["f/max"].to_float(),7.0);
+
+    EXPECT_EQ(d["g/mean"].to_float(),3.0);
```

Comment:
Just so we have test cases for some of the `int` arrays, I think adding `EXPECT_EQ(d["a/mean"].to_float(),3.0);` and `EXPECT_EQ(d["b/mean"].to_float(),2.0);` here would be beneficial.

Reply:
agreed, I added mean checks for all arrays